about/policy/license-disclaimer/index.md

```diff
@@ -24,18 +29,38 @@ The data and services provided by UGRC is licensed under a [Creative Commons Att
 ### License Attribution
 {: .text-left}
 
-UGRC requests that where possible and practical, **appropriate attribution** shall be made by users of this website, data, or any other services provided by UGRC with the following criteria:
+UGRC requests that wherever possible and practical, you include **appropriate attribution** when using data or services provided by UGRC in your maps, web applications, or other content. This includes providing credit to UGRC and indicating you made any changes to the data.
+
+For referenced data hosted by other organizations (such as data from the Utah Geological Survey), please contact the owner for attribution guidelines.
```

Comment:
**question**: should referenced be reference? Referenced or reference seems superfluous or what am I missing. Maybe I'm tired.

Reply:
Yeah, it's a little left-field. It's (trying not to say "referencing" here...) a tie-in to the definitions used in the sharing agreement, where "hosted" = data we host in the internal/Open SGID and "referenced" = other people's data referenced via the SGID Index or Open Data.